### ISSUE TITLE
mean acc of unseen is NaN

### ISSUE THREAD

**Issue by greathope** [bcmi/CaGNet-Zero-Shot-Semantic-Segmentation#10]
Hi, I checked the fasttext value and your pickled embedding value, it seems they does not match.  I directly  use word embeddings from fasttext, but the mean acc of unseen is NaN for my customed data, how to fix it? Thanks.

**Comment by zhangxgu** [comment]
Hi, @greathope .The word vectors of VOC and COCO-stuff is directly copied from SPNet, you can find them in their github. Also I have checked the word2vec of VOC, which is the same with the pretrained word files with normalization, except some differences between words 'airplane' and 'areoplane'.
Besides, mean acc of unseen is not NaN when you are running the finetuning step, by loading the configs with ``finetune''.

**Comment by greathope** [comment]
@zhangxgu yes, I directly run your voc12 and context, the mean acc of unseen is ok, but when I run finetuning on my customed data, the mean acc of unseen is nan...

**Comment by zhangxgu** [comment]
@greathope You may need to check the path of val data.

**Comment by qybing** [comment]
I have the same problem.Have you solved it？
